sentry-samples/sentry-samples-spring-boot-opentelemetry/src/test/kotlin/io/sentry/systemtest/PersonSystemTest.kt

```diff
@@ -44,4 +44,70 @@ class PersonSystemTest {
                 testHelper.doesTransactionContainSpanWithOp(transaction, "spanCreatedThroughSentryApi")
         }
     }
+
+    @Test
```

Comment:
TODO use `@BeforeTest` instead of `@Before` as it looks like it's not resetting between tests in the same test class.

Reply:
TODO copy this test over to other samples (e.g. jakarta version of this)

Reply:
Let's do this in a follow up PR